framework/bundles/org.jboss.tools.rsp.foundation.core/src/main/java/org/jboss/tools/rsp/foundation/core/transport/URLTransportCache.java

```diff
@@ -52,7 +52,7 @@
 	private static final Logger LOG = LoggerFactory.getLogger(URLTransportCache.class);
 
 	private static final int DEFAULT_CONNECT_TIMEOUT = 1 * 60 * 1000;
-	private static final int DEFAULT_READ_TIMEOUT = 2 * 60 * 1000;
+	private static final int DEFAULT_READ_TIMEOUT = (int)(0.1 * 60 * 1000);
```

Comment:
Is this intentional, to change the read timeout from 60 seconds to 6 seconds?

Reply:
no wasnt, correcting. thing that this was when I started writing tests and forget to discard it.